src/main/java/step4/domain/Cars.java

```diff
@@ -0,0 +1,61 @@
+package step4.domain;
+
+import step4.utils.RandomUtils;
+import step4.utils.StringUtils;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Cars {
+    private static final int MAX_BOUND = 10;
+
+    private final List<Car> cars;
+
+    private Cars(List<Car> cars) {
+        this.cars = cars;
+    }
+
+    public static Cars from(final String carNames) {
+        if (StringUtils.isBlank(carNames)) {
+            throw new IllegalArgumentException("자동차 이름을 입력하지 않았습니다.");
+        }
+        List<Car> cars = new ArrayList<>();
+        for (String name : carNames.split(",")) {
+            cars.add(Car.from(name));
+        }
+        return new Cars(cars);
+    }
+
+    public void move() {
+        for (Car car : cars) {
+            car.move(RandomUtils.getRandomNum(MAX_BOUND));
+        }
+    }
+
+    public List<String> getWinnerNames() {
+        int maxPosition = getMaxPosition();
+        List<String> winners = new ArrayList<>();
+        for (Car car : cars) {
+            addWinner(winners, car, maxPosition);
+        }
+        return winners;
+    }
+
+    private void addWinner(List<String> winners, final Car car, final int maxPosition) {
+        if (car.getPosition() >= maxPosition) {
+            winners.add(car.getName());
+        }
+    }
+
+    private int getMaxPosition() {
+        int maxPosition = 0;
+        for (int i = 0; i < cars.size() - 1; i++) {
```

Comment:
```suggestion
        for (Car car : cars) {
```
와 같이 foreach loop 를 사용하면 가독성 확보 및 index 를 신경쓰지 않을 수 있을 것 같아요~!

Reply:
for loop 안에서 인덱스를 이용해서 서로 position값을 비교하여 최대 Position값을 구하려고 했던 의도였는데 다시보니 가독성도 좋지 않고 stream을 이용하는게 훨씬 좋은거같아 수정하였습니다~